### PR TITLE
Merge back release 12.0.13

### PR DESCRIPTION
**Pull request by olamy**
- **deployAtEnd and a bit more memory for Maven process**
- **Updating to version 12.0.13**
- **next snapshot**

### CODE REVIEW

File: build/scripts/release-jetty.sh [hunk 1]
```diff
@@ -88,11 +88,11 @@ if [ ! -d "$ALT_DEPLOY_DIR" ] ; then
 fi
 
 # DEPLOY_OPTS="-Dmaven.test.failure.ignore=true"
-DEPLOY_OPTS="-DskipTests -Dasciidoctor.skip=false -Dmaven.build.cache.enabled=false"
+DEPLOY_OPTS="-DskipTests -Dasciidoctor.skip=false -Dmaven.build.cache.enabled=false -DdeployAtEnd=true -DretryFailedDeploymentCount=3 "
 # DEPLOY_OPTS="$DEPLOY_OPTS -DaltDeploymentRepository=intarget::default::file://$ALT_DEPLOY_DIR/"
 
 # Uncomment for Java 1.7
-export MAVEN_OPTS="-Xmx4g"
+export MAVEN_OPTS="-Xmx8g -Xms8g -client"
```
Comment (olamy) [inline build/scripts/release-jetty.sh]: @joakime only noticeable changes are in this script

Comment (sbordet) [reply build/scripts/release-jetty.sh]: @olamy remove `-client`, it does nothing since Java 9.

Comment (olamy) [reply build/scripts/release-jetty.sh]: done

**Review by sbordet — APPROVED** [review]

File: build/scripts/release-jetty.sh [hunk 2]
```diff
@@ -88,11 +88,11 @@ if [ ! -d "$ALT_DEPLOY_DIR" ] ; then
 fi
 
 # DEPLOY_OPTS="-Dmaven.test.failure.ignore=true"
-DEPLOY_OPTS="-DskipTests -Dasciidoctor.skip=false -Dmaven.build.cache.enabled=false"
+DEPLOY_OPTS="-DskipTests -Dasciidoctor.skip=false -Dmaven.build.cache.enabled=false -DdeployAtEnd=true -DretryFailedDeploymentCount=3 "
```
Comment (olamy) [inline build/scripts/release-jetty.sh:91]: @joakime do not be surprised during next release with deployAtEnd :)

**Pull request merged**